File: desktop/src/com/github/serivesmejia/eocvsketch/desktop/DesktopLauncher.java
package com.github.serivesmejia.eocvsketch.desktop;

import com.badlogic.gdx.backends.lwjgl.LwjglApplication;
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration;
import com.github.serivesmejia.eocvsketch.EOCVSketchApp;

public class DesktopLauncher {
	public static void main(String[] arg) {
		LwjglApplicationConfiguration config = new LwjglApplicationConfiguration();
		new LwjglApplication(new EOCVSketchApp(), config);
	}
}

File: ui/src/main/kotlin/com/github/serivesmejia/eocvsketch/ui/DesktopLauncher.kt
package com.github.serivesmejia.eocvsketch.ui

import com.badlogic.gdx.backends.lwjgl.LwjglApplication
import com.badlogic.gdx.backends.lwjgl.LwjglApplicationConfiguration

fun main() {
    val config = LwjglApplicationConfiguration()
    config.title = "EasyOpenCV Sketching v${EOCVSketchApp.VERSION}"

    LwjglApplication(EOCVSketchApp(), config)
}